frontends/web/src/api/exchanges.ts

```diff
@@ -17,19 +17,9 @@
 import { AccountCode } from './account';
 import { apiGet, apiPost } from '@/utils/request';
 
-export type ExchangeRegionList = {
-  regions: ExchangeRegion[];
-}
-
-export type ExchangeRegion = {
-  code: string;
-  isMoonpayEnabled: boolean;
-  isPocketEnabled: boolean;
-}
-
-export const getExchangesByRegion = (code: string) => {
-  return (): Promise<ExchangeRegionList> => {
-    return apiGet(`exchange/by-region/${code}`);
+export const getExchangeRegionCodes = () => {
+  return (): Promise<string[]> => {
```

Comment:
Nice that makes everything simpler.

Now that the `code` argument is gone, it doesn't need to return another function.

You could just return `return apiGet('exchange/region-codes');` here and pass it later to useLoad without calling the function:

`const regionCodes = useLoad(exchangesAPI.getExchangeRegionCodes);`